service/src/main/java/de/starwit/dto/FieldType.java

```diff
@@ -0,0 +1,23 @@
+package de.starwit.dto;
+
+public enum FieldType {
```

Comment:
Is there a way to access this enum? This would be really useful in frontend

Reply:
what is needed to access it? - we should talk about that